### PR TITLE
update quick start customization options

### LINKED ISSUE #2019: update quick start customization options
To be done after merging v0.30

- Give the three [customization options](https://docs.meilisearch.com/learn/getting_started/quick_start.html#customization) as bullet points (i.e. index settings, search parameters, instance options).
Communicating the different types of customization possible is more important than specifics of how to customize
- Rewrite intro paragraph so it's less redundant w/ following content

### PR DESCRIPTION
**Pull request by maryamsulemani97**
closes #2019

### CODE REVIEW

File: learn/getting_started/quick_start.md [hunk 1]
```diff
@@ -285,19 +285,15 @@ If you have multiple indexes, you can switch between them using the indexes drop
 
 ## Customization
 
-At this point, you can configure your Meilisearch instance and customize your index. When searching, you can use search parameters to refine your results.
+At this point, you can configure your entire Meilisearch instance, customize your index, and refine your results using:
```
Comment (guimachiavelli) [inline learn/getting_started/quick_start.md:288]: I think it's nice to indicate one instance can have multiple indexes.

```suggestion
At this point, you can configure your entire Meilisearch instance, customize your indexes, and refine your results using:
```

**Review by guimachiavelli — COMMENTED** [review]
One minor comment and we should be good to go 🤙

**Review by guimachiavelli — APPROVED** [review]
🏑

**Comment by maryamsulemani97** [comment]
bors merge

**Pull request merged**